test/integration/targets/sts_assume_role/tasks/main.yml

```diff
@@ -0,0 +1,382 @@
+---
+# tasks file for sts_assume_role
+
+- block:
+
+    # ============================================================
+    # TODO create simple ansible sts_get_caller_identity module
+    - blockinfile:
```

Comment:
I think we can just assume the availability of aws cli and use the same mechanism as hacking/aws_config/setup_iam.yml

Reply:
I did not assume that, because it was not available on CI test builder. But of course I can change that.